cg/cli/post_process/post_process.py

```diff
@@ -35,4 +35,25 @@ def post_process_sequencing_run(context: CGConfig, run_name: str, dry_run: bool)
     post_processing_service.post_process(run_name=run_name, dry_run=dry_run)
 
 
-post_process_group.add_command(post_process_sequencing_run)
+@post_process_group.command(name="all")
+@DRY_RUN
+@click.pass_obj
+def post_process_all_runs(context: CGConfig, dry_run: bool) -> None:
+    """Post-process all runs from the instruments."""
+    services: PostProcessingServices = context.post_processing_services
+    names_services: RunDirectoryNamesServices = context.run_directory_names_services
+    are_all_services_successful: bool = True
+    for service, name_service in [(services.pacbio, names_services.pacbio)]:
+        try:
+            service.post_process_all(
+                run_directory_names=name_service.get_run_names(), dry_run=dry_run
+            )
+        except Exception as error:
```

Comment:
I agree that the error handling is too much, but in the <do-something>-all commands we always except all Exceptions, otherwise it will halt production if there is a, for example, Pydantic error or something else

Reply:
Then I do not see the point in wrapping our errors at all. The whole point is to be able to perform clearer error handling?